101-sqlazure-ragr/azuredeploy.parameters.json

```diff
@@ -3,34 +3,28 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
```

Comment:
sorry, to be clear - remove parameters that are not needed by CI, empty strings will be interpreted as valid